llvm/docs/AMDGPUUsage.rst

```diff
@@ -1170,6 +1170,23 @@ The AMDGPU backend implements the following LLVM IR intrinsics.
 
   :ref:`llvm.set.fpenv<int_set_fpenv>`             Sets the floating point environment to the specifies state.
 
+  llvm.amdgcn.readfirstlane                        Provides direct access to v_readfirstlane_b32. Returns the value in
+                                                   the lowest active lane of the input operand. Currently 
+                                                   implemented for i16, i32, float, half, bf16, v2i16, v2f16 and types 
+                                                   whose sizes are multiples of 32-bit.
+
+  llvm.amdgcn.readlane                             Provides direct access to v_readlane_b32. Returns the value in the 
+                                                   specified lane of the first input operand. The second operand 
+                                                   specifies the lane to read from. Currently implemented
+                                                   for i16, i32, float, half, bf16, v2i16, v2f16 and types whose sizes
```

Comment:
Should probably try to avoid repeating all the same information on all 3 intrinsics

Reply:
Updated